project/ScalafixBuild.scala

```diff
@@ -137,9 +134,10 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
         "nightly" -> version.value,
         "scalameta" -> scalametaV,
         scalaVersion,
-        "supportedScalaVersions" -> supportedScalaVersions,
+        "supportedScalaVersions" -> cliScalaVersions,
```

Comment:
This does not feel right.
* Semantics of "supported" are unclear
* This build info should be in `cli` or `rules` rather than `core` (which does not cross-publish to Scala 3 yet)
 
Ffor backward compatibility with build tools, I'd rather keep it there for now.